<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hcmgis_split_form</class>
 <widget class="QDialog" name="hcmgis_split_form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>188</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Split Layers</string>
  </property>
  <widget class="QDialogButtonBox" name="BtnOKCancel">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>150</y>
     <width>156</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="LblInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Layer</string>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="CboInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>24</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LblOutput_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>54</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Unique field</string>
   </property>
  </widget>
  <widget class="QgsFieldComboBox" name="CboField">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LblOutput_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Folder</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LinOutputFolder">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOutputFolder">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>BtnOKCancel</sender>
   <signal>accepted()</signal>
   <receiver>hcmgis_split_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnOKCancel</sender>
   <signal>rejected()</signal>
   <receiver>hcmgis_split_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
